hs_tools_resource/models.py

```diff
@@ -230,9 +234,56 @@ def remove(cls, element_id):
 class ToolIcon(AbstractMetaDataElement):
     term = 'ToolIcon'
     value = models.CharField(max_length=1024, blank=True, default="")
+    data_url = models.TextField(blank=True, default="")
+
+    @classmethod
+    def _validate_tool_icon(cls, url):
+        try:
+            response = requests.get(url, verify=False)
+        except:
+            raise ValidationError("Failed to read data from given url.")
```

Comment:
@zhiyuli use of ```except:```, i.e., a bare except without specifying an exception type, is highly discouraged. You will hide the actually exception being raised. You should log and show the exception message instead.